test/powershell/Modules/Microsoft.PowerShell.Core/Job.Tests.ps1

```diff
@@ -0,0 +1,76 @@
+Describe "Job Cmdlet Tests" -Tag "CI" {
```

Comment:
as it turns out the cmdlets actually do run on windows - the jobs themselves don't work, but the creation/retrieval cmdlets do work. Also, currently the job creation fails on windows which is why these tests are marked as pending